docs/reference/api.mdx

```diff
@@ -81,3 +81,27 @@ GitHub: https://github.com/rarimo/issuer
 | Mainnet  | https://api.robotornot.rarimo.com |
 |----------|----------------------------|
 | Testnet  | https://api.robotornot.mainnet-beta.rarimo.com |
+
+## PolygonID x Rarimo
```

Comment:
Put all Polygon ID stuff into a single document instead of splitting API and contracts. Use PoH reference as an example:

https://docs.rarimo.com/reference/proof-of-humanity

Reply:
Also include a link to the demo page